mixer/pkg/config/crd/store.go

```diff
@@ -91,7 +91,7 @@ func waitForSynced(ctx context.Context, informers map[string]cache.SharedInforme
 // Store offers store.StoreBackend interface through kubernetes custom resource definitions.
 type Store struct {
 	conf         *rest.Config
-	ns           map[string]bool
+	ns           string
```

Comment:
cc @ayj If I remember correctly this was added so the CRD controller can watch a list of namespaces. Is this behavior preserved by the changes in the PR?

Reply:
Yes, this changes the behavior, from list of namespaces to a single namespace.
Justification: so far I don't see any use cases to watch multiple namespaces. e2e tests specifies a single namespace, and others (like cluster-wide tests and release version) watch all namespaces. The new code supports both, and no one needs to list up multiple namespaces so far. I think the existing scheme was overly generalized.

Added this note to the PR description.